lib/cli/index.ts

```diff
@@ -139,6 +144,7 @@ yargs
               .option(SELENIUM_PORT, seleniumPort)
               .option(STANDALONE, standaloneOption)
               .option(STANDALONE_NODE, standaloneNodeOption)
+              .option(GRID_NODE, gridNodeOption)
```

Comment:
We will add this with .option and .alias. https://github.com/angular/webdriver-manager/blob/master/lib/cli/index.ts#L178

Reply:
ok - done